proto/osmosis/incentives/pot.proto

```diff
@@ -0,0 +1,42 @@
+syntax = "proto3";
+package osmosis.incentives;
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/duration.proto";
+import "google/protobuf/timestamp.proto";
+import "cosmos/base/v1beta1/coin.proto";
+
+option go_package = "github.com/c-osmosis/osmosis/x/incentives/types";
+
+enum LockQueryType {
+  option (gogoproto.goproto_enum_prefix) = false;
+
+  ByDuration = 0; // locks which has more than specific duration
+  ByTime = 1; // locks which are started before specific time
+}
+
+message DistrCondition {
+  LockQueryType lock_query_type = 1; // type of lock, ByLockDuration | ByLockTime
+  string denom = 2; // lock denom
+  // condition for lock duration, only valid if positive
+  google.protobuf.Duration duration = 3
+    [(gogoproto.stdduration) = true, (gogoproto.nullable) = false, (gogoproto.moretags) = "yaml:\"duration\""];
+  // condition for lock start time, not valid if unset value
+  google.protobuf.Timestamp timestamp = 4
+    [(gogoproto.stdtime) = true, (gogoproto.nullable) = false, (gogoproto.moretags) = "yaml:\"timestamp\""];
+}
+
+message Pot {
+  uint64 id = 1; // unique ID of a Pot
+  // distribute condition of a lock which meet one of these conditions
+  DistrCondition distribute_to = 2 [(gogoproto.nullable) = false];
+  repeated cosmos.base.v1beta1.Coin coins = 3
+    [(gogoproto.nullable) = false, (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coins"]; // can distribute multiple coins
+  // condition for lock start time, not valid if unset value
```

Comment:
Can this comment be updated to disambiguate relative to the DistrCondition's comment?

Reply:
Ok will update